test/formatters/apheleia-ft.el

```diff
@@ -260,7 +260,7 @@ environment variable, defaulting to all formatters."
                  in-file)
                (list stdout-buffer stderr-file)
                nil
-               (cdr command)))
+               (cdr `(,command))))
```

Comment:
Isn't the `cdr` of a list with one element always going to be nil?

Reply:
Hm yes you're right, this is a leftover change from a previous impl (I tried to use `sh-shell` and `indent-tabs-mode`) so I can remove it now I'm using a flat list